AsteroidGame/Objects/GameObjectException.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AsteroidGame.Objects
+{
+    class GameObjectException : Exception
+    {
+        public override string Message { get { return "Неверно задан параметр \"" + ParamName + "\" в объекте \"" + this.ToString() + "\"\nОписание: " + description; } }
+        public string ParamName { get { return paramName; }  }

```

Comment:
Можно (нужно) упростить. 

```suggestion
        public string ParamName => paramName;

```